sonar-db/src/main/java/org/sonar/db/user/UserDao.java

```diff
@@ -70,7 +70,9 @@ public UserDto selectUserById(SqlSession session, long userId) {
    * Used by the Governance plugin
    */
   public List<UserDto> selectByIds(DbSession session, Collection<Long> ids) {
-    return DatabaseUtils.executeLargeInputs(ids, new SelectByIds(mapper(session)));
+    return executeLargeInputs(
+      ids,
+      partitionOfLogins -> session.getMapper(UserMapper.class).selectByIds(partitionOfLogins));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1612)